content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -0,0 +1,97 @@
+---
+title: AWS Manual Setup Guide
+kind: guide
+description: "Steps for manually setting up the Datadog AWS Integration"
+further_reading:
+- link: "https://docs.datadoghq.com/integrations/amazon_web_services/"
+  tag: "Documentation"
+  text: "AWS Integration"
+- link: "https://docs.datadoghq.com/serverless/libraries_integrations/forwarder/"
+  tag: "Documentation"
+  text: "Datadog Forwarder Lambda function"
+- link: "https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination/"
+  tag: "Guide"
+  text: "Send AWS service logs with the Datadog Kinesis Firehose destination"
+- link: "https://docs.datadoghq.com/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/"
+  tag: "Guide"
+  text: "AWS CloudWatch metric streams with Kinesis Data Firehose"
+- link: "https://www.datadoghq.com/blog/aws-monitoring/"
+  tag: "Blog"
+  text: "Key metrics for AWS monitoring"
+- link: "https://www.datadoghq.com/blog/monitoring-ec2-instances-with-datadog/"
+  tag: "Blog"
+  text: "How to monitor EC2 instances with Datadog"
+- link: "https://www.datadoghq.com/blog/monitoring-aws-lambda-with-datadog/"
+  tag: "Blog"
+  text: "Monitoring AWS Lambda with Datadog"
+- link: "https://www.datadoghq.com/blog/cloud-security-posture-management/"
+  tag: "Blog"
+  text: "Introducing Datadog Cloud Security Posture Management"
+- link: "https://www.datadoghq.com/blog/datadog-workload-security/"
+  tag: "Blog"
+  text: "Secure your infrastructure in real time with Datadog Cloud Workload Security"
+- link: "https://www.datadoghq.com/blog/announcing-cloud-siem/"
+  tag: "Blog"
+  text: "Announcing Datadog Security Monitoring"
+---
+
+## Overview
+
+Use this guide to manually set up the Datadog [AWS Integration][1].
+
+Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
+
+## Setup
```

Comment:
@buraizu one major structural change, is that I think we should have the `Access keys: Used for GovCloud or China only` manual setup instructions in the same page. I think it makes sense to have  "Role delegation" and 
"Access keys (GovCloud or China Only)" as tab options with different instructions. Both of these are manual setups. I don't think it makes sense to have another guide for manual setup for govcloud and china.

Reply:
Thanks for pointing this out! Updated in ac38017